operations/helm/charts/mimir-distributed/templates/gateway/gateway-dep.yaml

```diff
@@ -25,7 +25,7 @@ spec:
         {{- if .Values.useExternalConfig }}
         checksum/config: {{ .Values.externalConfigVersion }}
         {{- else }}
-        checksum/config: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
+        checksum/config: {{ include (print $.Template.BasePath "/mimir-config.yaml") . | sha256sum }}
```

Comment:
WDYT about moving these common annonations into a named template (`{{- define "mimir.podAnnonations" -}} ...`)? Similar to `mimir.podLabels`. That same template an do the checksum calculation and even accept the `.Values.gateway.podAnnotations` as an arg an inject them.

Reply:
sounds fair, probably safe to do now since we have the golden record checks, let me see

Reply:
added